examples/generalized_linear_models/GLM-robust.ipynb

```diff
@@ -16,7 +16,7 @@
     "\n",
```

Comment:
<p>Note that here you're not passing a custom family. You're telling Bambi to use the family named "t", which is a built-in family since version 0.6.0</p><p> </p><p>Custom families are things created with <code>Family()</code>  class</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>got it, thanks for clarifying that</p>